src/deploy-cromwell-on-azure/Configuration.cs

```diff
@@ -31,6 +31,7 @@ public class Configuration : UserAccessibleConfiguration
 
     public abstract class UserAccessibleConfiguration
     {
+        public string AksNodeResourceGroupName { get; set; }
```

Comment:
The only thing I would add would be to validate in `Deployer.cs` that this is not set for Upgrades (to follow the pattern of most other "initial deployment only" properties). There are already other properties where that's done, it should be in the same place.

Will this also be done in the CoA deployer?